### PR TITLE
Bpk 1215 split pod into subspecs

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: Example/Tests/BPKColorTest.m [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 #import <XCTest/XCTest.h>
 
-#import <Backpack/Backpack.h>
+#import <Backpack/Color.h>
```
Comment (matthewdavidson) [inline Example/Tests/BPKColorTest.m]: To confirm, this is the breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# Unreleased
+
+*Breaking:*
+
+- Moved colors to a subspec in `Backpack/Color`.
```
Comment (matthewdavidson) [inline CHANGELOG.md]: Side note, we should add dates to the changelog when we release (YYYY-MM-DD)

File: Example/Tests/BPKColorTest.m (hunk 1 above)
Comment (k0nserv) [reply Example/Tests/BPKColorTest.m]: Yeah

File: CHANGELOG.md (hunk 2 above)
Comment (k0nserv) [reply CHANGELOG.md]: Sure thing :)

**Comment by k0nserv** [comment]
I'm thinking we release this as `0.1.0`

**Pull request merged**